### PR TITLE
Add more languages support.

### PR DESCRIPTION
**Pull request by lakshayrohila**
Does these things:-

1. Adds fonts-unifont in DISTRO_PKGS_SPECS-ubuntu-jammy (main jammypup64 iso) (might not look very good for few languages, but atleast removes few TOFUs).
2. Adds **fonts-noto-cjk** and **fonts-noto-core** to **nls.sfs** for **JammyPup64**.
3. Adds **gtk3-nocsd**.

### CODE REVIEW

File: woof-distro/x86_64/ubuntu/jammy64/DISTRO_PKGS_SPECS-ubuntu-jammy [hunk 1]
```diff
@@ -63,7 +63,7 @@ yes|bison|bison|exe>dev,dev,doc,nls||deps:yes
 yes|blueman|blueman|exe,dev,doc,nls||deps:yes
 yes|bluez|bluez,bluez-obexd|exe,dev,doc,nls||deps:yes
 no|boehm-gc|libgc1,libgc-dev|exe,dev,doc,nls||deps:yes
-no|busybox||exe
+yes|busybox||exe
```
Comment (dimkr) [inline woof-distro/x86_64/ubuntu/jammy64/DISTRO_PKGS_SPECS-ubuntu-jammy]: This conflicts with the busybox petbuild

File: woof-distro/x86_64/ubuntu/jammy64/DISTRO_PKGS_SPECS-ubuntu-jammy [hunk 2]
```diff
@@ -678,7 +681,7 @@ no|xsoldier|xsoldier|exe,dev>null,doc,nls
 yes|xtrans|xtrans-dev|exe>dev,dev,doc,nls||deps:yes
 no|xvidcore|libxvidcore4,libxvidcore-dev|exe,dev,doc,nls
 yes|xz|xz-utils,liblzma5,liblzma-dev|exe,dev,doc,nls||deps:yes
-no|yad||exe
+yes|yad||exe
```
Comment (dimkr) [inline woof-distro/x86_64/ubuntu/jammy64/DISTRO_PKGS_SPECS-ubuntu-jammy]: This conflicts with the yad petbuild

Comment (lakshayrohila) [reply woof-distro/x86_64/ubuntu/jammy64/DISTRO_PKGS_SPECS-ubuntu-jammy]: Sorry, forgot that I had added those. Will remove them now.

**Comment by dimkr** [comment]
Binary files should not be added to woof-CE like this - either add the Ubuntu fonts-unifont package, or add a petbuild that builds this package from source. Binary files make a git repo bigger and slower to clone, because changes are not tracked normally.

**Comment by lakshayrohila** [comment]
> Binary files should not be added to woof-CE like this - either add the Ubuntu fonts-unifont package, or add a petbuild that builds this package from source. Binary files make a git repo bigger and slower to clone, because changes are not tracked normally.

The binary file added is just ~11MBs, and is also TTF. Adding it to woof-code directory should make it usable with all pups. Please correct me if I am wrong, will correct it then.

Thanks

**Comment by dimkr** [comment]
> The binary file added is just ~11MBs, and is also TTF.

That's a lot! The repo will grow by 11 MB every time you change this file.

**Comment by dimkr** [comment]
(Plus, you're forcing this 11 MB on **any** Puppy built using woof-CE - adding fonts-unifont to jammy64 will solve your problem in a way that doesn't increase the size of the repo, doesn't collide with upstream distro font packages and doesn't force anything on other Puppy builds)

**Comment by lakshayrohila** [comment]
> (Plus, you're forcing this 11 MB on **any** Puppy built using woof-CE - adding fonts-unifont to jammy64 will solve your problem in a way that doesn't increase the size of the repo, doesn't collide with upstream distro font packages and doesn't force anything on other Puppy builds)

Done the requested. Removed binary file and added fonts-unifont to in DISTRO_PKGS_SPECS-ubuntu-jammy.

**Comment by dimkr** [comment]
Do you want this PR to be merged as-is or prefer to leave out some unrelated things like the get_libreoffice change?

**Comment by lakshayrohila** [comment]
I removed desk_icon_theme_neon and gnome-keyring which seemed extremely unnecessary. get_libreoffice, ghostscript, gparted and htop can be kept. What do you think?

**Comment by dimkr** [comment]
> What do you think?

Up to you, I don't mind these changes. Nobody is currently working on jammy64, so you can do pretty much whatever you want to it.

**Comment by lakshayrohila** [comment]
I think these can be kept. Especially gparted and get_libreoffice as most of the users will like to have those.

**Comment by dimkr** [comment]
Squashed to prevent the 11 MB binary file from making it into the git history and merged. 🎉

**Pull request merged**